code/modules/shuttle/arrivals.dm

```diff
@@ -7,6 +7,7 @@
 	height = 15
 	dir = WEST
 	port_angle = 180
+	preferred_direction = WEST

```

Comment:
I know what preferred_direction does, I added it. It determines which direction the ship wants to travel in while in transit space. You've just placed the transit port in a strange and random place, that's not shared by any other shuttle.